pymeasure/experiment/parameters.py

```diff
@@ -115,6 +122,20 @@ def convert(self, value):
 
         return value
 
+    def _cli_help_fields(self):
+        message = self.name
+        for field in self._help_fields:
+            if isinstance(field, str):
+                field = ["{} is".format(field), field]
+
+            if hasattr(self, field[1]) and getattr(self, field[1]) is not None:
+                prefix = field[0]
+                value = getattr(self, field[1])
+                message += ", {} {}".format(prefix, value)
```

Comment:
Here I'd also suggest using f-strings:

```suggestion
                message += f", {prefix} {value}"
```